### PR TITLE
Add setMeetingQuality functionality

### PR DESCRIPTION
**Pull request by adamweeks**
# Pull Request Template

## Description

Adds the `setMeetingQuality` functionality to the meeting object.

Fixes # https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-133791

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Meetings
- [ ] Test B

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 1]
```diff
@@ -2867,4 +2870,38 @@ export default class Meeting extends StatelessWebexPlugin {
...
+
+      return Promise.reject(new Error(errorMessage));
+    }
+
+    // Set the overall quality level
+    this.mediaProperties.setQualityLevel(level);
+
+    const qualityActions = [];
+
+    const {receiveAudio, receiveVideo, sendVideo} = this.mediaProperties.mediaDirection;
```
Comment (lalli-flores) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:2895]: Can you add a comment on what this is doing, and how it's different from setting the media properties? Or explain the steps a bit more

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js [hunk 2]
```diff
@@ -695,7 +699,60 @@ describe('plugin-meetings', () => {
...
+        beforeEach(() => {
+          mediaDirection = {
+            receiveAudio: true, receiveVideo: true, receiveShare: false, sendVideo: true
+          };
+          meeting.setRemoteQualityLevel = sinon.stub().returns(Promise.resolve());
+          meeting.setLocalVideoQuality = sinon.stub().returns(Promise.resolve());
+          meeting.mediaProperties.mediaDirection = mediaDirection;
+        });
+
+        it('should have #setMeetingQuality', () => {
```
Comment (lalli-flores) [inline packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js:715]:
```suggestion
        it('should exist as a meeting object method', () => {
```

Comment (adamweeks) [reply packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js:715]: This is just to keep consistent with the rest of the tests in this suite.

**Comment by adamweeks** [comment]
Test failures are consistent with other integration failures with `The peer connection is currently negotiation an offer. Please wait for a few seconds and try again.`. Merging.

**Pull request merged**